core/Updates/2.13.1.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Common;
+use Piwik\Updater;
+use Piwik\Updates;
+
+/**
+ * Update for version 2.13.1.
+ */
+class Updates_2_13_1 extends Updates
+{
+    /**
+     * Here you can define one or multiple SQL statements that should be executed during the update.
+     * @return array
+     */
+    static function getSql()
+    {
+        $optionTable = Common::prefixTable('option');
+        $removeEmptyDefaultReportsSql = "delete from $optionTable where option_name like '%defaultReport%' and option_value=''";
```

Comment:
Should be "delete from `$optionTable`", otherwise will fail on MySQL if there's no table prefix.